views/admin/tpl/tab_descriptor.tpl

```diff
@@ -0,0 +1,26 @@
+[{*
+* Shop System Plugins:
+* - Terms of Use can be found under:
+* https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+* - License can be found under:
+* https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+*
+*}]
+
+[{include file="headitem.tpl" title="GENERAL_ADMIN_TITLE"|oxmultilangassign}]
+
+<form name="transfer" id="transfer" action="[{$oViewConf->getSelfLink()}]" method="post">
+    [{$oViewConf->getHiddenSid()}]
+    <input type="hidden" name="oxid" value="[{$oxid}]">
+    <input type="hidden" name="cl" value="[{$controller}]">
+</form>
+
+[{if $data}]
+    [{$data[0]|html_entity_decode}]
```

Comment:
Why do you need to use `html_entity_decode` here?

Reply:
👍 It's not necessary. I removed it.